### PR TITLE
feat: RUST_REGEX supports more messages

### PR DESCRIPTION
**Pull request by qexat**
:rocket: :fire: :stars:

### CODE REVIEW

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]
Should use a word boundary instead of a space for `blazingly fast` imo

**Comment by qexat** [comment]
> Should use a word boundary instead of a space for `blazingly fast` imo

how so? it is already using `\b`...?

**Comment by qexat** [comment]
do you mean `\s`?

**Comment by Lunarmagpie** [comment]
> 
> how so? it is already using `\b`.

Like `\bblazingly\bfast\b`

**Comment by CircuitSacul** [comment]
> how so? it is already using \b...?

I think lunar means in between the words (e.x blazingly\bfast)

**Comment by qexat** [comment]
> > how so? it is already using `\b`.
> 
> Like `\bblazingly\bfast\b`

that's kinda weird to me tbh, like any word boundary between blazingly and fast? :thinking:

**Comment by qexat** [comment]
i'll change it anyway

**Review by Lunarmagpie — APPROVED** [review]

**Comment by MithicSpirit** [comment]
Pretty sure that `\b` doesn't actually match any characters and only boundaries between characters, so now this doesn't match anything.

**Comment by MithicSpirit** [comment]
imo undo that change. just leave it as a space. If you really want to match different types of spaces you could match on space, nbsp, zwsp, tab, newline, underscore, hyphen, etc.

File: mcodingbot/plugins/reactions.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from mcodingbot.utils import Plugin
 
-RUST_REGEX = re.compile(r"\brust\b", flags=re.I)
+RUST_REGEX = re.compile(r"(\brust\b|\bblazingly fast\b)", flags=re.I)
```
Comment (trag1c) [inline mcodingbot/plugins/reactions.py:8]:
```suggestion
RUST_REGEX = re.compile(r"(\brust\b|\bblazingly\s+fast\b)", flags=re.I)
```

**Comment by qexat** [comment]
@Lunarmagpie @CircuitSacul can one of you approve to merge please?

**Review by trag1c — APPROVED** [review]

**Pull request merged**